### PR TITLE
Fix message pass through test

### PR DESCRIPTION
**Pull request by ShimShtein**

### CODE REVIEW

File: app/consumers/job_creator_consumer.rb [hunk 1]
```diff
@@ -1,12 +1,15 @@
...
+require 'dispatcher'
 
 class JobCreatorConsumer < Racecar::Consumer
   subscribes_to Notifications::INCOMING_TOPIC
 
   def process(kafka_message)
+    message_value = kafka_message.value
+    Rails.logger.debug("Received message: #{message_value}")
+
     message = Message.from_json(kafka_message.value)
```
Comment (adamruzicka) [inline app/consumers/job_creator_consumer.rb]: Why not reuse the previously extracted `message_value` here?

Comment (ShimShtein) [reply app/consumers/job_creator_consumer.rb]: That was the intention. Forgot about it, sorry.

**Comment by ShimShtein** [comment]
@adamruzicka, fixed and rebased.

**Comment by adamruzicka** [comment]
Thanks @ShimShtein !

**Pull request merged**